service/ui_plugin_manifest.go

```diff
@@ -14,7 +14,13 @@ func (uis *UIServer) GetManifest(w http.ResponseWriter, r *http.Request) {
 	project := vars["project_id"]
 	revision := vars["revision"]
 
-	version, err := model.VersionFindOne(model.BaseVersionByProjectIdAndRevision(project, revision))
+	projectId, err := model.FindIdForProject(project)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("error getting project ID for project '%s': %s", project, err),
```

Comment:
err.Error()

Reply:
this is actually super redundant, i'm going to just use err.Error()